Content/install/liquibase-installation-troubleshooting.htm

```diff
@@ -6,17 +6,37 @@
     </head>
     <body>
         <h1 style="white-space: nowrap;">Installation Troubleshooting</h1>
-        <p>If you cannot run the <code>liquibase --help</code> command successfully, perform the following steps:</p>
+        <p>If you cannot run the <code>liquibase --version</code> command successfully, perform the following steps:</p>
```

Comment:
I love this validation & troubleshooting section! It's informative and covers everything I can think of that can be problematic during initial use of the CLI.